addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -40,6 +41,14 @@ def _default_default_message(self):
     title_color = fields.Char(default="#FFFFFF", help="Default title color of the channel once open")
     button_background_color = fields.Char(default="#875A7B", help="Default background color of the Livechat button")
     button_text_color = fields.Char(default="#FFFFFF", help="Default text color of the Livechat button")
+    concurent_session = fields.Boolean()
```

Comment:
Could be better to give a more meaningful name such as `limit_concurrent_session` or something that suggests that it's a boolean. Also, I believe that `concurrent` takes two "r".

Reply:
Resolved, but there is still a [typo](https://dictionary.cambridge.org/dictionary/english/concurrent). The same applies to the XML field descriptions.